src/components/flyout/flyout.tsx

```diff
@@ -50,6 +61,10 @@ export interface EuiFlyoutProps
    * Defines the width of the panel
    */
   size?: EuiFlyoutSize;
+  /**
+   * Customize the padding around the content of the flyout header, body and footer.
```

Comment:
Just a nit-picky consistency thing
```suggestion
   * Customize the padding around the content of the flyout header, body and footer
```